docs/debugging.rst

```diff
@@ -60,6 +60,21 @@ When an application is built, ``flatpak-builder`` automatically
 creates a ``.Debug`` extension. This can be disabled with the ``no-debuginfo``
 option.
 
+To install the ``.Debug`` extension created locally, export the build
+to a repo by passing ``--repo repo`` to ``flatpak-builder`` when
+building. Then add the repo folder as a local remote::
+
+  $ flatpak remote-add --user --no-gpg-verify test-debug file:///path/to/repo
```

Comment:
if you have built with `flatapk-builder --install` you don't need this, nor even `--repo`.